app/models/middleware_diagnostic_report.rb

```diff
@@ -0,0 +1,5 @@
+class MiddlewareDiagnosticReport < ApplicationRecord
+  self.table_name = "middleware_diagnostic_reports"
```

Comment:
Is this necessary? I think the table name is derived, no?

Reply:
@karelhala was this addressed?

Reply:
Well if you look at https://github.com/ManageIQ/manageiq-providers-hawkular/pull/72 I removed the table name there so the table name is same for every diagnostic report which can be derived from this model.

Reply:
@karelhala this is only needed if you need to override the default, you can still get the `#table_name` from the model if you don't want to look at the class name or file name.  Please remove.

Reply:
@agrare all done.